index.js

```diff
@@ -1,17 +1,61 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [],
+    this.length = 0
+  }
 
-  add(item) {}
+  add(item) {
+      this.items.push(item);
+      this.items.sort((a, b) => a - b);
+      this.length ++;
+    
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (typeof this.items[pos] !== 'number'){
```

Comment:
in this case, the goal was to find pos >=  this.length.
But the way you are checking also works, but its not as easy to parse and read what is happening.

easier:
```js
if (pos >= this.length)
```